test/parallel/test-cli-node-cli-manpage-env-vars.mjs

```diff
@@ -21,16 +21,6 @@ assert(manpageEnvVarNames.size > 0,
        'Unexpectedly not even a single env variable was detected when scanning the `doc/node.1` file'
 );
 
-// TODO(dario-piotrowicz): add the missing env variables to the manpage and remove this set
-//                         (refs: https://github.com/nodejs/node/issues/58894)
-const knownEnvVariablesMissingFromManPage = new Set([
```

Comment:
Since the variable is being removed its use below needs to be removed as well :slightly_smiling_face: